package/gluon-config-mode-autoupdater-freifunk/Readme.md

```diff
@@ -0,0 +1,35 @@
+# gluon-config-mode-autoupdater-freifunk
+
+The package gluon-config-mode-autoupdater-freifunk provides an
+ alternative to both gluon-config-mode-autoupdater and gluon-web-autoupdater.
+By replacing both packages, gluon-config-mode-autoupdater-freifunk it cannot be installed in combination: Both
+have to be missing from site.mk.
+ 
+Users are asked for consent before enabling automatic updates. This follows (Freifunk's 
+Memorandum of understanding)[https://github.com/freifunk/MoU]. The question is asked using radio-buttons without any defaults. It is neither opt-in nor opt-out.
+
+To maintain compatibility with existing setups and keep defaults, enabling updates and giving consent is handled differently.
+
+* By giving or revoking consent, users can enable or disable automatic updates
+* If automatic-updates are enabled, but consent is not given, 
+a message is shown telling, that consent was given out of band. By that, defaults from gluon-config-mode-autoupdater 
+and gluon-web-autoupdater are implemented.
```

Comment:
Thus settings from gluon-config-mode-autoupdater 
and gluon-web-autoupdater will be inherited.